cadc-file-validate/README.md

```diff
@@ -0,0 +1,74 @@
+# cadc-file-validate 0.1
+
+Command line tool to ensure validity of the information stored in the Inventory Database and what is served from a Storage Adapter.  
+
+This is intended to run periodically at any site to determine what changes, if any, are to be made to either the Storage Inventory database, or
+the storage system.
+
+## Configuration
+
+A file called `cadc-file-validate.properties` should be available from the `/config` directory.  All properties in the file are loaded and set
+as System properties:
+
+```java
+Properties properties = new Properties();
+properties.load("/config/cadc-file-validate.properties");
+System.setProperties(properties);
+```
+
```

Comment:
Please look at some of the other README files and try to follow the same style... they aren't all consistent yet (there's an issue for that) so be careful which you follow.

Also, the storage adapter implementation will require it's own set of properties).... I wonder if the file here should be named java-system.properties so it's clear that it contains config from several different parts.

Reply:
This is based off of the Minoc README.  I've updated it a bit.

Reply:
The file contains properties for the storage adapter, the validator, and the database.  We could have a `storage-inventory.properties`, too, but these properties exist together because of the validator.  I'm not sure what the best solution is for the name of the file.